src/libstd/sys/unix/ext/net.rs

```diff
@@ -1467,6 +1467,701 @@ impl IntoRawFd for UnixDatagram {
     }
 }
 
+/// A structure representing a Unix domain seqpacket socket server.
+///
+/// # Examples
+///
+/// ```no_run
+/// #![feature(unix_socket)]
+/// use std::thread;
+/// use std::os::unix::net::{UnixSeqpacket, UnixSeqpacketListener};
+///
+/// fn handle_client(socket: UnixSeqpacket) {
+///     // ...
+/// }
+///
+/// let listener = UnixSeqpacketListener::bind("/path/to/the/socket").unwrap();
+///
+/// // accept connections and process them, spawning a new thread for each one
+/// for socket in listener.incoming() {
+///     match socket {
+///         Ok(socket) => {
+///             /* connection succeeded */
+///             thread::spawn(|| handle_client(socket));
+///         }
+///         Err(err) => {
+///             /* connection failed */
+///             break;
+///         }
+///     }
+/// }
+/// ```
+#[unstable(feature = "unix_socket", issue = "0")]
```

Comment:
The feature name has to be new for tidy to be happy - `unix_socket_seqpacket` or whatever seems fine.